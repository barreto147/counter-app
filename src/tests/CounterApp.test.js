import CounterApp from "../CounterApp";
import { shallow } from "enzyme";

describe("CounterApp Pruebas", () => {
  test("CounterApp debe hacer match con snapshot", () => {
    const wraper = shallow(<CounterApp />);
    expect(wraper).toMatchSnapshot();
  });
  test("CounterApp debe mostrar el valor por defecto 100", () => {
    const wraper = shallow(<CounterApp value={100} />);
    const counterText = wraper.find("h2").text().trim();
    expect(counterText).toBe("100");
  });
});
