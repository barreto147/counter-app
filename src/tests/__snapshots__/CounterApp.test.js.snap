// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CounterApp Pruebas CounterApp debe hacer match con snapshot 1`] = `
<Fragment>
  <h1>
    CounterApp
  </h1>
  <h2>
    10
  </h2>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
</Fragment>
`;

exports[`CounterApp Pruebas CounterApp debe mostrar el valor por defecto 100 1`] = `
<Fragment>
  <h1>
    CounterApp
  </h1>
  <h2>
    100
  </h2>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
</Fragment>
`;
